src/mono/browser/runtime/types/internal.ts

```diff
@@ -242,6 +242,27 @@ export type RuntimeHelpers = {
     forceDisposeProxies: (disposeMethods: boolean, verbose: boolean) => void,
     dumpThreads: () => void,
     mono_wasm_print_thread_dump: () => void,
+
+    //used by hybrid globalization module
+    stringToUTF16: (dstPtr: number, endPtr: number, text: string) => void,
+    stringToUTF16Ptr: (str: string) => VoidPtr,
+    utf16ToString: (startPtr: number, endPtr: number) => string,
+    utf16ToStringLoop: (startPtr: number, endPtr: number) => string,
+    localHeapViewU16: () => Uint16Array,
+    setU16_local: (heap: Uint16Array, ptr: number, value: number) => void,
+    setI32: (offset: MemOffset, value: number) => void,
+
+    mono_wasm_change_case_invariant: (src: number, srcLength: number, dst: number, dstLength: number, toUpper: number) => VoidPtr;
```

Comment:
could we create `GlobalizationHelpers` and move it there ?

Reply:
Instead of using `RuntimeHelpers`? We could.

Reply:
just for the hybrid members